### ISSUE TITLE
ui5-tooling-transpile babelConfig is not applied

### ISSUE THREAD

**Issue by C3ntraX** [ui5-community/ui5-ecosystem-showcase#864]
**Describe the bug**
Ich habe die ui5 cli von 2.x auf 3.x hochgezogen und musste auch von ui5-middleware-transpile auf ui5-tooling-transpile-task umsteigen. Ich habe jedoch festgestellt, dass meine babelConfig seitdem nicht mehr ausgelesen und verwendet wird.

**To Reproduce**
My cofiguration:
```
  - name: ui5-tooling-transpile-task
    afterTask: replaceVersion
    configuration:
      excludePatterns:
        - "scripts/"
      babelConfig: &babelConfig
        plugins:
          - 
            - "@babel/plugin-transform-computed-properties"
            - loose: true
          - 
            - "@babel/plugin-transform-for-of"
            - assumeArray: true
          -
            - "babel-plugin-transform-async-to-promises"
            - inlineHelpers: true
          - 
            - "@babel/plugin-proposal-optional-chaining"
            - assumeArray: true
        presets:
          - 
            - "@babel/preset-env"
            - targets: "> 0.25%, not dead, Chrome > 75"
```

- If you do a build, you will see, that for ecample 'await' or '??=' do not transform anymore

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by petermuessig** [comment]
@C3ntraX can you please share the build log? The should be an information in the log what babel configuration is being used, e.g.: `Using inline Babel configuration from ui5.yaml...`

https://github.com/ui5-community/ui5-ecosystem-showcase/blob/main/packages/ui5-tooling-transpile/lib/util.js#L387

If you turn on the debug mode of the tooling extension with `debug: true` in the configuration section and running the whole build with verbose, you should also see the generated babel config: 

https://github.com/ui5-community/ui5-ecosystem-showcase/blob/main/packages/ui5-tooling-transpile/lib/util.js#L379

**Comment by petermuessig** [comment]
Found it, issue is fixed with `ui5-tooling-transpile` version `3.2.6`
